### ISSUE TITLE
Unable to filter transactions report

### ISSUE THREAD

**Issue by stephhjacq** [Automattic/woocommerce-payments#2507]
When trying to apply a filter to view transactions for last month, the screen appears blank and a JS error is thrown.

### Steps to reproduce

Go to Payments > Transactions
Show > Advanced Filters
Add filter > Date
Between > try to select a date - error is thrown: TypeError: t.onMonthPreviewed is not a function

All Payments menu items cannot be accessed until a hard refresh is done. 

<img width="1409" alt="Screen Shot 2021-07-21 at 13 35 36" src="https://user-images.githubusercontent.com/29288537/126557304-e258ca0e-0115-47fa-a471-f194b5c288be.png">

WooCommerce 5.5.1
WooCommerce Payments 2.7.0
WordPress 5.8

**Comment by kalessil** [comment]
There was a similar report (related to the dates filtering) https://github.com/Automattic/woocommerce-payments/issues/1632

**Comment by tpaksu** [comment]
Sorry, added the fix and labels to the wrong issue :) Removed them.

**Comment by aprea** [comment]
Looks like this was fixed upstream here - https://github.com/WordPress/gutenberg/pull/31751

This fix was included in Gutenberg 10.9.

I installed & activated the latest Gutenberg plugin locally (version 11.1.0) and was unable to reproduce this issue.

We'll need to wait until this fix is included in WordPress core.

As a temporary workaround, affected users can install the Gutenberg plugin.

WP core is currently at 5.8. The next major release is 5.9 and [is scheduled](https://wordpress.org/about/roadmap/) for Dec 2021.

I can introduce a temporary fix in `woocommerce-admin` until the proper fix is released in WP core but I'd rather not as we'd have to revert it at a later time.

@Automattic/gamma, any thoughts on above? I'm unsure of how WP core releases work. Is it possible to petition for a patch release to include the upstream Gutenberg fix?
